src/allmydata/test/test_magic_folder.py

```diff
@@ -1113,11 +1176,11 @@ def Alice_conflicts_with_Bobs_last_downloaded_uri():
             return d2
 
         d.addCallback(lambda ign: Alice_conflicts_with_Bobs_last_downloaded_uri())
-        d.addCallback(lambda ign: self._check_downloader_count('objects_downloaded', 4))
-        d.addCallback(lambda ign: self._check_downloader_count('objects_conflicted', 0))
+        d.addCallback(lambda ign: self._check_downloader_count('objects_downloaded', 4, magic=self.bob_magicfolder))
+        d.addCallback(lambda ign: self._check_downloader_count('objects_conflicted', 1, magic=self.bob_magicfolder))
```

Comment:
Yeah ... not sure about that mechanism of "testing" -- but also I think this "test_alice_bob" should go away because it's waaaay too big (but then we need to answer the question: "did we cover all the cases this happens to be covering"...probably best as a separate PR?)